### ISSUE TITLE
What should the time loss look like, why lr=0.00002

### ISSUE THREAD

**Issue by Liujingxiu23** [rishikksh20/TFGAN#5]
1. Why you set lr=0.00002, have you tried other settings, is lr=0.00002 the best?

2. In the paper, the time loss is an import point,  but in my training, the time loss seems not decline.
I use my own dataset , sample rate of 16k, the mels are extracted using scripts in tacotron2. Training is still in progress, and the generated wavs become cleaner with more steps.  And I started the trainning of disc at 100k steps. 
<img width="207" alt="g_loss" src="https://user-images.githubusercontent.com/49260794/106086648-866bda00-615d-11eb-9274-0ce49d48f8d8.PNG">
@rishikksh20  what is the time loss in your trainning?

**Comment by rishikksh20** [comment]
* you can change `lr` to `0.0002` . I just used my another codebase as base for this implementation that's why I think thats carry over.
* Time loss is the loss function calculated in time domain instead of mel freq., it's the sum of energy loss, time loss and phase loss.

**Comment by Liujingxiu23** [comment]
@rishikksh20  thank you for your quick reply, I did experiment on the same condition with lr=0.0002 and 0.00002, though training is going on, generated wavs of lr=0.0002 is better using the model with the same epoch.

**Comment by krantiparida** [comment]
@rishikksh20 Hi, Thanks for your effort and time in making this code available. I have a question regarding the behaviour of STFT loss as observed in the plot shared by @Liujingxiu23. It can be seen that the STFT loss increases when the adversarial loss kicks in. I also observe similar performance. Can you please suggest what might be happening here? Ideally, the loss curve should decrease.
